tests/clar.c

```diff
@@ -440,21 +446,24 @@ void
 clar_test_shutdown(void)
 {
 	clar_print_shutdown(
+		_clar.evaluation_duration,
 		_clar.tests_ran,
-		(int)_clar_suite_count,
-		_clar.total_errors
+		_clar.suites_ran,
+		_clar.total_errors,
+		_clar.total_skipped
 	);
 
 	clar_unsandbox();
 }
 
 int
-clar_test(int argc, char **argv)
+clar_test(void)
 {
 	int errors;
+	double start = git__timer();
 
-	clar_test_init(argc, argv);
 	errors = clar_test_run();
+	_clar.evaluation_duration = git__timer() - start;
```

Comment:
As said, I'd like to retain the current signature. If doing so, you could still easily put the timer-code into `clar_test_run`.